modulus/experimental/metrics/climate/geometric.py

```diff
@@ -0,0 +1,283 @@
+# Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Optional, List
+
+import torch
+from torch import nn
+
+import torch_harmonics as harmonics
+from torch_harmonics.quadrature import clenshaw_curtiss_weights
+
+
+# TODO (era5_wind) double check if polar optimization has an effect - we use 5 here by default
+class GeometricLpLoss(nn.Module):
+    """Geometric Lp loss"""
+
+    def __init__(
+        self,
+        inp_size: List[int],
+        p: float = 2.0,
+        size_average: bool = False,
+        reduction: bool = True,
+        absolute: bool = False,
+        squared: bool = False,
+        pole_mask: int = 0,
+        jacobian: str = "s2",
+        quadrature_rule: str = "naive",
+    ):  # pragma: no cover
+        super(GeometricLpLoss, self).__init__()
+
+        self.p = p
+        self.inp_size = inp_size
+        self.reduction = reduction
+        self.size_average = size_average
+        self.absolute = absolute
+        self.squared = squared
+        self.pole_mask = pole_mask
+
+        if jacobian == "s2":
+            jacobian = torch.sin(
+                torch.linspace(0, torch.pi, self.inp_size[0])
+            ).unsqueeze(1)
+        else:
+            jacobian = torch.ones(self.inp_size[0], 1)
+
+        if quadrature_rule == "naive":
+            dtheta = torch.pi / self.inp_size[0]
+            dlambda = 2 * torch.pi / self.inp_size[1]
+            dA = dlambda * dtheta
+            quad_weight = dA * jacobian
+        elif quadrature_rule == "clenshaw-curtiss":
```

Comment:
Shouldn't this be `clenshaw-curtis` without the double s at the end?

Reply:
I ended up removing this file